core/src/internal/rpc/test.rs

```diff
@@ -74,6 +71,57 @@ fn connect_attempt() {
     handle.join().unwrap();
 }
 
+
+// //this test launches a chrome tab so commented out so it won't run with usual tests
```

Comment:
Same "commented out code" comments as above.

Reply:
will put in new branch